YAFC/Widgets/DataGrid.cs

```diff
@@ -62,7 +62,7 @@ public class DataGrid<TData> where TData : class {
         private void BuildHeaderResizer(ImGui gui, DataColumn<TData> column, Rect rect) {
             switch (gui.action) {
                 case ImGuiAction.Build:
-                    var center = rect.X + rect.Width * 0.5f;
```

Comment:
here we are putting the parenthesis back (see one of my previous comments were  the unnecessary ones were removed). Is sounds conflicting?

Reply:
I think this one should be preferred, but is just my personal opinion. My head is faster detecting the the parentheses than parsing operator precedence ;)

Reply:
> sounds conflicting?

The rules were unmuted one by one. Some of them overwrite each other, so it indeed could happen that some parentheses were put back.